python/processing/algfactory.py

```diff
@@ -320,10 +322,12 @@ class ProcessingAlgFactory():
     RASTER_LAYER = "RASTER_LAYER",
     VECTOR_LAYER = "VECTOR_LAYER",
     MESH_LAYER = "MESH_LAYER",
+    POINT_CLOUD_LAYER = "MESH_LAYER",
```

Comment:
```suggestion
    POINT_CLOUD_LAYER = "POINT_CLOUD_LAYER",
```

Reply:
Ouch, sorry. I really should not code with that nasty cold.